fiboa_cli/datasets/fieldscapes_corsica_2022.py

```diff
@@ -0,0 +1,139 @@
+
+# Unique identifier for the collection
+ID = "fieldscapes_corsica_2022"
+# Title of the collection
+TITLE = "Field boundaries for Corsica (Fieldscapes)"
+# Description of the collection. Can be multiline and include CommonMark.
+DESCRIPTION = """ The dataset contains field boundaries for the Corsica."""
+# Bounding box of the data in WGS84 coordinates
+BBOX = [8.684215733091785, 41.726866147253986, 9.55961872699482, 42.808624416493835]
+
+# Provider name, can be None if not applicable, must be provided if PROVIDER_URL is provided
+PROVIDER_NAME = "Republique Francaise - Geoservices"
+# URL to the homepage of the data or the provider, can be None if not applicable
+PROVIDER_URL = "https://data.geopf.fr/telechargement/download/RPG/RPG_2-0__SHP_LAMB93_R94_2022-01-01/RPG_2-0__SHP_LAMB93_R94_2022-01-01.7z.001"
+# Attribution, can be None if not applicable
+ATTRIBUTION = "Institut national de l'information géographique et forestière (IGN), Agence de Services et de Paiement (ASP)"
+
+# License of the data, either
+# 1. a SPDX license identifier (including "dl-de/by-2-0" / "dl-de/zero-2-0"), or
+LICENSE = "CC-BY-2.0"
+# 2. a STAC Link Object with relation type "license"
+# LICENSE = {"title": "CC-BY-4.0", "href": "https://creativecommons.org/licenses/by/4.0/", "type": "text/html", "rel": "license"}
+
+# Map original column names to fiboa property names
+# You also need to list any column that you may have added in the MIGRATION function (see below).
+COLUMNS = {
+    "ID_PARCEL": "id",
+    "SURF_PARC": "area",
+    "CODE_GROUP": "crop_id",
+    "CODE_CULTU": "crop_name",    
+    "geometry": "geometry"
+}
+
+# Add columns with constant values.
+# The key is the column name, the value is a constant value that's used for all rows.
+ADD_COLUMNS = {
+    "determination_datetime": "2022-01-01T00:00:00Z"
+}
+
+# A list of implemented extension identifiers
+EXTENSIONS = []
+
+# Functions to migrate data in columns to match the fiboa specification.
+# Example: You have a column area_m in square meters and want to convert
+# to hectares as required for the area field in fiboa.
+# Function signature:
+#   func(column: pd.Series) -> pd.Series
+COLUMN_MIGRATIONS = {
+    
+}
+
+# Filter columns to only include the ones that are relevant for the collection,
+# e.g. only rows that contain the word "agriculture" but not "forest" in the column "land_cover_type".
+# Lamda function accepts a Pandas Series and returns a Series or a Tuple with a Series and True to inverse the mask.
+COLUMN_FILTERS = {
+    
+}
+
+# Custom function to migrate the GeoDataFrame if the other options are not sufficient
+# This should be the last resort!
+# Function signature:
+#   func(gdf: gpd.GeoDataFrame) -> gpd.GeoDataFrame
+MIGRATION = None
+
+# Schemas for the fields that are not defined in fiboa
+# Keys must be the values from the COLUMNS dict, not the keys
+MISSING_SCHEMAS = {
+    "required": [], # i.e. non-nullable properties
+    "properties": {
```

Comment:
schemas mssing for crop_name and crop_id

Reply:
Do we have to do this for all the countries?

Reply:
If there's no extension, yes. That's why I recommended an extension.